game.rb

```diff
@@ -0,0 +1,80 @@
+class Game
+  attr_accessor :board
+  attr_accessor :win
+
+  def initialize
+    @win = false
+    @board = { A1: ' ', A2: ' ', A3: ' ', B1: ' ', B2: ' ', B3: ' ', C1: ' ', C2: ' ', C3: ' ' }
+  end
+
+  def draw
+    puts ' -------------'
+    puts "3| #{board[:A3]} | #{board[:B3]} | #{board[:C3]} |"
+    puts ' -------------'
+    puts "2| #{board[:A2]} | #{board[:B2]} | #{board[:C2]} |"
+    puts ' -------------'
+    puts "1| #{board[:A1]} | #{board[:B1]} | #{board[:C1]} |"
+    puts ' -------------'
+    puts '   A   B   C'
+  end
+
+  def winner(row)
+    return if row.any? { |s| s.strip.empty? }
```

Comment:
To rozwiązanie działa, natomiast można je uprościć organizując pola na planszy w tak zwany `magiczny kwadrat` http://mathforum.org/alejandre/magic.square/adler/adler.whatsquare.html
http://www.cs.pomona.edu/~kim/CSC051S06/Lectures/Lecture23/Lecture23_1.html
Takie podejście też sprawiłoby, że jeśli - hipotetycznie - kiedyś chcielibyśmy zwiększyć planszę do gry to sprawdzanie warunku zwycięstwa nie będzie dużo trudniejsze niż dla kwadratu 3x3.